nni/common/graph_utils.py

```diff
@@ -724,6 +726,8 @@ def _build_graph(self):
                     unique_name = module_name
                     if use_count > 0:
                         unique_name = module_name + '.%d' % use_count
+                        self.reused_module.add(unique_name)
+                        self.reused_module.add(module_name)
```

Comment:
though `use_count > 0`, `node` is not necessarily reused module. you can double check when `use_count` is increased by 1

Reply:
`use_count>1` happens only when there are more than one separated subgraphs that have the same `module_name`. Note that this module is a leaf module of the model, therefore, this leaf module is used more than once in the model graph(reused).